### PR TITLE
Update basic-bonfires.json

### PR DESCRIPTION
**Pull request by Legend28469**
Added Array.slice() to Bonfire Chunky Monkey's helpful links as I believe it is equally helpful as Array.push() in that specific challenge.

### CODE REVIEW

File: seed/challenges/basic-bonfires.json [hunk 1]
```diff
@@ -505,6 +505,7 @@
       ],
       "MDNlinks": [
         "Array.push()"
```
Comment (bugron) [inline seed/challenges/basic-bonfires.json]: @Legend28469 please, add a comma at the end of this line, thanks.

**Comment by bugron** [comment]
@Legend28469 thanks for the PR. Please read our [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) and [How To Create A Pull Request for Free Code Camp](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/How-To-Create-A-Pull-Request-for-Free-Code-Camp) and next time follow the instructions from this last article, thanks.

**Comment by bugron** [comment]
@Legend28469 what's your progress on applying changes?

**Comment by BerkeleyTrue** [comment]
This PR is closed due to inactivity. 

Please feel free to re-open when it is ready to be merged.

Happy Coding!

**Pull request closed without merging**